### ISSUE TITLE
Train neural network failed with exit code 1

### ISSUE THREAD

**Issue by dominikacecylia** [modernmt/modernmt#597]
I am trying to run a command `./mmt create en pl /workspace -e eupengine --train-steps 20000`
on a dataset from the EU parliament dataset, but it crashes with the following error message:

`ERROR Unexpected exception: Command 'fairseq-train /opt/mmt/runtime/alldata/tmp/training/data_generated --save-dir /opt/mmt/runtime/alldata/tmp/training/_temp_train/nn_model --task mmt_translation --user-dir /opt/mmt/build/mmt-4.5.4.jar/mmt --share-all-embeddings --no-progress-bar --tensorboard-logdir /opt/mmt/runtime/alldata/tmp/training/_temp_train/tensorboard_logdir --dataset-impl mmap --max-update 2000 --arch transformer_mmt_base --clip-norm 0.0 --label-smoothing 0.1 --attention-dropout 0.1 --dropout 0.3 --weight-decay 0.0 --criterion label_smoothed_cross_entropy --optimizer adam --adam-betas (0.9, 0.98) --log-interval 100 --lr 0.0005 --lr-scheduler inverse_sqrt --min-lr 1e-09 --warmup-init-lr 1e-07 --warmup-updates 4000 --max-tokens 3072 --update-freq 4 --save-interval-updates 1000 --keep-interval-updates 10 --keep-last-epochs 10' failed with exit code 1`

This doesn't tell me much about the issue, I first tried running it with a larger dataset but it failed with the same error. Now testing this smaller chunk but still same error.
I checked out the other suggestions in the issues but didn't help me solve mine. Not sure if this is related to the version of ModernMT? using the modernmt/master image.
